v21.2/change-data-capture-overview.md

```diff
@@ -23,6 +23,26 @@ The main feature of CDC is the changefeed, which targets an allowlist of tables,
 
 See [Ordering Guarantees](use-changefeeds.html#ordering-guarantees) for detail on CockroachDB's at-least-once-delivery-guarantee as well as explanation on how rows are emitted.
 
+## How does a changefeed work?
+
+When a changefeed is started on a node, that node becomes the aggregator node for the changefeed job (CRDB Node 2 in the diagram). The aggregator node acts as an administrator: keeping track of all other nodes during job execution and the changefeed work as it completes. In the unlikely event that the changefeed’s aggregator node were to fail during the job, the aggregator role will move to a different node and the changefeed will restart from the last checkpoint.
```

Comment:
> When a changefeed is started on a node, that node becomes the aggregator node for the changefeed job (CRDB Node 2 in the diagram). The aggregator node acts as an administrator

Internally, we have "change aggregator" processors; and a single change frontier processor. Aggregators run on all nodes, while frontier is a single node.  Perhaps we can "rename" frontier processor to be "changefeed coordinator"?  And use "change aggregator" to mean "the thing that runs on each node"?

Also, note re picture: THe coordinator node runs "both" processors (so it functions as both an aggregator and a coordinator).

Reply:
> The aggregator node acts as an administrator: keeping track of all other nodes during job execution and the changefeed work as it completes. In the unlikely event that the changefeed’s aggregator node were to fail during the job, the aggregator role will move to a different node

I think it is important to emphasize that this functionality is applicable only to enterprise changefeeds.  Core style changefeeds do not even have a job.  The caller is expected to handle any hickups -- so, if the node where coordinator is running is restarted, the SQL connection used in core style changefeeds will return an error -- the caller is responsible for restarts.

Another thing to note is that the picture shows enterprise style changefeed -- (kafka).  So, perhaps this text makes sense, provided that we make it clear that this whole thing is for "enterprise".

It might also make sense to have another picture -- very similar to the one you have, but specifically for core style changefeed where each aggregator sends data back to the coordinator, and the coordinator sends the data back to the caller through SQL conneciton.